crates/nu-command/tests/commands/run_external.rs

```diff
@@ -213,6 +213,17 @@ fn external_command_not_expand_tilde_with_quotes() {
     )
 }
 
+#[test]
+fn external_command_receives_raw_binary_data() {
+    Playground::setup("external command receives raw binary data", |dirs, _| {
+        let actual = nu!(cwd: dirs.test(), pipeline(r#"0x[deadbeef] | hexdump -C"#));
+        assert_eq!(
+            actual.out,
+            r#"00000000  de ad be ef                                       |....|00000004"#
```

Comment:
I don't know if the padding of `hexdump`'s output is stable and set in stone, so this might make this test less stable. Also I am not sure if it is available by default on Windows systems were our tests are also run. I wouldn't be a fan of completely disabling the test on Windows as your PR introduces semantics that should be general.

Reply:
Yeah I wasn't a fan of using hexdump in this test either, but I wasn't able to figure another way. I'm not sure of any default external program that can process hex bytes. Piping into `^cat` was outputting different bytes than it received so I didn't think it'd be stable to test that either.

Reply:
What about removing padding stuff in `hexdump` to make our test reliable.

Generally like this:
```rust
assert_eq!(
    actual.out.replace(" ", ""),
   r#"00000000deadbeef|....|00000004"#
)
```